client/src/pages/AboutUs.js

```diff
@@ -79,6 +81,8 @@ const supporterLogos = [
   workableLogo,
   zendeskLogo,
   zeplinLogo,
+  dataDogLogo,
+  pagerDuty,
```

Comment:
Can we display them in order, like add datadog before datasaur logo and pager duty before papertrail ?